extensions/arc/runtime/src/main/java/io/quarkus/arc/lookup/LookupIfProperty.java

```diff
@@ -30,7 +30,7 @@
  *  }
  *
  *  {@literal @ApplicationScoped}
- *  class ServiceBar {
```

Comment:
Could you also fix the javadoc in `LookupUnlessProperty`?

Reply:
Sure thing!